.circleci/config.yml

```diff
@@ -1,37 +1,24 @@
-version: 2
+version: 2.1    
 jobs:
   build:
     docker:
-      - image: ux3d/gltf-blender-io
+      - image: p3din/gltf-blender-io
```

Comment:
is this line meant to be checked in or should we rebuild it?

Reply:
I setup my fork to build automatically on docker hub (`p3din/gltf-blender-io`). I think ideally someone with enough permission should set up builds on docker hub for the main repo, maybe `khronosgroup/gltf-blender-io`. The main thing about automated builds is that they will then continously get updated blender 2.80 builds.

As mentioned I did try to include the docker image build on CircleCI, but that adds another 5 min. to build times there,

Reply:
The automated update is a good idea! I will set that up for `ux3d/gltf-blender-io` as well, then we can continue using that url.

Reply:
Should be good now :)